### DISCUSSION TITLE
看介绍说用到了mobx，可是我没有发现mobx的痕迹，能指点下吗？

### DISCUSSION THREAD

**Discussion by ghost** [alibaba/formily#1192] [Q&A 2.x] [answered]
你好，
我在该项目主页，发现 有mobx的标签，
![image](https://user-images.githubusercontent.com/6580897/113408180-7680a880-93e1-11eb-8c82-2b40a12803aa.png)

在知乎 https://www.zhihu.com/question/328612405/answer/1630737866  也看到作者说 用到了mobx

可是，我在 https://github.com/alibaba/formily/blob/master/package.json 里没有找到mobx的痕迹

请问是找的方法不对吗？

方便指点下吗

谢谢

**Comment by atzcl** [comment, marked as answer]
https://github.com/alibaba/formily/blob/formily_next/packages/reactive/docs/guide/index.md

**Reply by ghost** [reply]
谢谢， 原来重新造了个轮子
